### PR TITLE
Feature/product

### PR DESCRIPTION
**Pull request by nberongoy**
## Description

- Description

## References

- [JIRA card](https://identifi.atlassian.net/browse/JIRA-CARD)

## Types of changes

- Bug fix
- Non-breaking change
- Breaking change

## Affected Components

- <Component Here>

### CODE REVIEW

**Comment by nberongoy** [comment]
feature/product

File: modules/ProductComponent/ProductCard.tsx [hunk 1]
```diff
@@ -64,22 +64,35 @@ const ProductCard: FC<IProductCard> = ({ product }) => {
...
-          <Box mt="1" fontWeight="bold" as="h4" lineHeight="tight" isTruncated>
-            {productData.name}
+          <Box
+            mt="1"
+            fontWeight="bold"
+            as="h4"
+            lineHeight="tight"
+            onClick={(): void => {
+              dispatch(setSelectedProduct(product));
+            }}
```
Comment (vmc08) [inline modules/ProductComponent/ProductCard.tsx:87]: you can omit function typings @nberongoy

File: modules/ProductComponent/Products.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ export interface IProductEdge {
 }
 
 const Products: FC = ({}) => {
-  const { data } = useQuery(FETCH_PRODUCTS, { variables: { first: 9999 } });
+  const { data } = useQuery(FETCH_PRODUCTS, { fetchPolicy: 'cache-and-network', variables: { first: 9999 } });
```
Comment (vmc08) [inline modules/ProductComponent/Products.tsx:25]: cache-and-network I believe is the default fetchPolicy cc @nberongoy

File: modules/ProductComponent/DeleteProdcut.tsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ const ProductDeleteModal: FC<IDeleteProductsProps> = ({ isOpen, onClose, product
 
   const toast = useToast();
 
-  const onDelete = async (): Promise<void> => {
+  const onDelete = async (): Promise<any> => {
```
Comment (vmc08) [inline modules/ProductComponent/DeleteProdcut.tsx:28]: you can omit typings here @nberongoy

**Review by vmc08 — COMMENTED** [review]
Looking good so far, just added some comments you can improve next PRs.

File: modules/ProductComponent/Products.tsx (hunk 2 above)
Comment (nberongoy) [reply modules/ProductComponent/Products.tsx:25]: it was cache first for the default one @vmc08 

image

**Pull request merged**